<?xml version="1.0"?>
<robot name="MemristorBig">
    <ros2_control name="HardwareControl" type="system">
        <hardware>
            <plugin>mep3_hardware::RobotHardwareInterface</plugin>
            <param name="kp_linear">90.0</param>
            <param name="ki_linear">1.0</param>
            <param name="kd_linear">2.0</param>

            <param name="kp_angular">110.0</param>
            <param name="ki_angular">1.0</param>
            <param name="kd_angular">0.0</param>

            <param name="update_rate">100.0</param>
        </hardware>
        <joint name="left_motor">
            <state_interface name="position" />
            <command_interface name="velocity" />
        </joint>
        <joint name="right_motor">
            <state_interface name="position" />
            <command_interface name="velocity" />
        </joint>
    </ros2_control>
</robot>
